src/System.Net.Http.WinHttpHandler/tests/FunctionalTests/WinHttpHandlerTest.cs

```diff
@@ -19,6 +19,9 @@ namespace System.Net.Http.WinHttpHandlerFunctional.Tests
     // to separately Dispose (or have a 'using' statement) for the handler.
     public class WinHttpHandlerTest
     {
+        // TODO: This is a placeholder until GitHub Issue #2383 gets resolved.
+        private const string SlowServer = "http://httpbin.org/drip?numbytes=1&duration=1&delay=40&code=200";
```

Comment:
Because I plan to change this soon.  "SlowServer" will become several different uri that exhibit different forms of "slow-ness".  There can be delays before any HTTP response headers are sent at all.  And there can be delays for just reading the response body but no delays for the headers themselves.  So, I didn't want this in common yet because I didn't want many people starting to use it yet.